### ISSUE TITLE
[ISSUE] - Industrial Primitive Blast Furnace not working

### ISSUE THREAD

**Issue by FallDark** [Frontiers-PackForge/CosmicFrontiers#253]
**To Reproduce**
try let it work

**Expected behavior**

**Screenshots/Crash Reports**
![20241025231440](https://github.com/user-attachments/assets/e468ad17-e584-4b6e-8fc7-b160f61a7131)
![20241025231500](https://github.com/user-attachments/assets/a41a7c70-e6d1-4605-b72a-8239c372e9b6)
![20241025231949](https://github.com/user-attachments/assets/46d73600-2503-4e0f-b839-bfb829b15516)
![20241025231529](https://github.com/user-attachments/assets/30ea9da5-e071-48d6-a3ff-b9ab8ca7e3ef)

**Desktop (please complete the following information):**
 - OS: Win 10
 - Pack Version lastest

**Comment by LynnRom** [comment]
I can confirm the problem exists as I'm currently unable to make any of the recipes work. If it's useful I grabbed the data from the block in a creative world:
![image](https://github.com/user-attachments/assets/49b6770e-b3f4-4b71-9e30-f034c3b54585)

edit: to add, this is with the bronze inputs and outputs; bronze fluid input (hatch) is filled with 4B of Creosote; bronze item input (bus) is filled with one pack of Iron Ingots and one pack of Coke in gem form (the one you get by just cooking up the vanilla coal item); and the bronze item output (bus) is, of course, empty. Sorry for the one hundred edits, I'm bad at this.

**Comment by Ghostipedia** [comment]
I will look into this next weekend, as well as a variety of other bugs recently reported. It was working when i tested it so much have been a last minute update to GTCEU:Modern that caused the problem.

**Comment by LynnRom** [comment]
I'm new to github so I'm not sure how to properly link and cite code (assuming there is a more proper way than this) but I'd wager a guess that the bug has to do with issue 2108 of gregtech and its associated commit (ed7a599): https://github.com/GregTechCEu/GregTech-Modern/pull/2108

TL;DR: They changed how you declare the max parellel recipes steam multiblocks can run so that it isn't hardcoded to 8.

**Comment by Ghostipedia** [comment]
Fixed with GTM 1.5.4 and CosmicCore 0.4.1 - Will be in 0.5.6 Cosmic Frontiers which is currently in progress 👍
